src/ucs/sys/sys.c

```diff
@@ -744,6 +744,21 @@ ucs_status_t ucs_tcpip_socket_create(int *fd_p)
     return UCS_OK;
 }
 
+ucs_status_t ucs_unix_socketpair_create(int *fd_1_p, int *fd_2_p)
```

Comment:
maybe pass AF and SOCK_xx as params?

Reply:
should we do the same for `ucs_tcpip_socket_create`?

Reply:
yes

Reply:
done